tests/e2e/performance/test_pod_reattachtime.py

```diff
@@ -62,85 +140,128 @@ def test_pvc_reattach_time_performance(self, pvc_factory, teardown_factory):
         node_one = worker_nodes_list[0]
         node_two = worker_nodes_list[1]
 
-        # Create a PVC
-        accessmode = constants.ACCESS_MODE_RWX
-        if self.interface == constants.CEPHBLOCKPOOL:
-            accessmode = constants.ACCESS_MODE_RWO
-        pvc_obj = pvc_factory(
-            interface=self.interface,
-            access_mode=accessmode,
-            status=constants.STATUS_BOUND,
-            size="15",
-        )
-
-        # Create a pod on one node
-        logging.info(f"Creating Pod with pvc {pvc_obj.name} on node {node_one}")
-
-        helpers.pull_images(constants.PERF_IMAGE)
-        pod_obj1 = helpers.create_pod(
-            interface_type=self.interface,
-            pvc_name=pvc_obj.name,
-            namespace=pvc_obj.namespace,
-            node_name=node_one,
-            pod_dict_path=constants.PERF_POD_YAML,
-        )
+        time_measures = []
+        for sample_index in range(1, samples_num + 1):
+            # Create a PVC
+            accessmode = constants.ACCESS_MODE_RWX
+            if self.interface == constants.CEPHBLOCKPOOL:
+                accessmode = constants.ACCESS_MODE_RWO
+            pvc_obj = pvc_factory(
+                interface=self.interface,
+                access_mode=accessmode,
+                status=constants.STATUS_BOUND,
+                size="15",
+            )
 
-        # Confirm that pod is running on the selected_nodes
-        logging.info("Checking whether pods are running on the selected nodes")
-        helpers.wait_for_resource_state(
-            resource=pod_obj1, state=constants.STATUS_RUNNING, timeout=120
-        )
+            # Create a pod on one node
+            logging.info(f"Creating Pod with pvc {pvc_obj.name} on node {node_one}")
 
-        pod_name = pod_obj1.name
-        pod_path = "/mnt"
+            helpers.pull_images(constants.PERF_IMAGE)
+            pod_obj1 = helpers.create_pod(
```

Comment:
as above, make sure the pod was created

Reply:
Will do the same ( try: ...except, with exception)